src/lib.rs

```diff
@@ -96,6 +96,18 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone {
             self.map.insert(key, (value, time::SteadyTime::now()));
         }
     }
+/// Remove a key/value pair from cache
```

Comment:
The accumulator uses it to update the responses then add them back to the cache. It's also called via the sentinel when a message has been validated. If 'get' returned a mutable reference we could probably update from that, but the logic changes, since if it gets updated it should be moved to the back of the cache and not all gets are mutating. That would also leave validated entries to be popped when the cache is full or ttl expires. I think having remove is the easiest option.